src/main/java/abbyy/cloudsdk/v2/client/Serializer.java

```diff
@@ -78,11 +83,20 @@ private static String getCollectionValues(Iterable<?> iterable) {
         List<String> values = new ArrayList<>();
         for (T element : array) {
             String value = element.toString();
+
+            if (element.getClass().isEnum()) {
+                value = getEnumValue(value);
+            }
+
             values.add(value);
         }
         return String.join(",", values);
     }
 
+    private static String getEnumValue(String value) {
+        return Character.toLowerCase(value.charAt(0)) + value.substring(1);
```

Comment:
Why should we do this?

Reply:
Such conversion is used in serialization algorithm in .NET implementation. For some reason, if request parameter is a collection of enums, each value of collection is serialized with first lowercase letter. To save uniformity of solution among different platforms, I've decided to maintain this approach